src/main/java/com/google/devtools/build/lib/syntax/FuncallExpression.java

```diff
@@ -236,7 +236,7 @@ public FuncallException(String msg) {
 
   private final int numPositionalArgs;
 
-  FuncallExpression(Expression function, ImmutableList<Argument.Passed> arguments) {
+  public FuncallExpression(Expression function, ImmutableList<Argument.Passed> arguments) {
```

Comment:
The lib.syntax package has changed a lot in the last few months, so you'll need to sync up.